admin/class-social-facebook-form.php

```diff
@@ -1,273 +0,0 @@
-		echo '<ul id="user_admin">';
-		$this->show_user_admins( $nonce );
-		echo '</ul>';
-		echo '</div>';
-
-		unset( $nonce );
-
-		$this->add_button(
-			array(
-				'url'   => '#TB_inline?width=600&height=350&inlineId=add_facebook_admin',
-				'value' => $button_text,
-				'class' => 'thickbox',
-				'title' => $button_text,
-			)
-		);
-
-		return $this;
-	}
-
-	/**
-	 * Show input field to set a facebook apps as an admin
-	 *
-	 * @return $this
-	 */
-	private function manage_app_as_admin() {
-		echo '<div class="clear"></div><br />';
-		Yoast_Form::get_instance()->textinput( 'fbadminapp', __( 'Facebook App ID', 'wordpress-seo' ) );
-
-		return $this;
-	}
-
-	/**
-	 * Loop through the fb-admins to parse the output for them
-	 *
-	 * @param string $nonce Nonce string.
-	 */
-	private function show_user_admins( $nonce ) {
-		foreach ( $this->fb_admins as $admin_id => $admin ) {
-			echo $this->get_admin_link( $admin_id, $admin, $nonce );
-		}
-	}
-
-	/**
-	 * Parsing the link that directs to the admin removal
-	 *
-	 * @param string $admin_id Facebook admin ID.
-	 * @param string $nonce    Nonce string.
-	 *
-	 * @return string
-	 */
-	private function admin_delete_link( $admin_id, $nonce ) {
-		return add_query_arg(
-			array(
-				'delfbadmin' => esc_attr( $admin_id ),
-				'nonce'      => $nonce,
-			),
-			admin_url( $this->admin_url . '#top#facebook' )
-		);
-	}
-
-	/**
-	 * Adding a button to the button property
-	 *
-	 * @param array $args Arguments data array.
-	 */
-	private function add_button( $args ) {
-		$args = wp_parse_args(
-			$args,
-			array(
-				'url'   => '',
-				'value' => '',
-				'class' => '',
-				'id'    => '',
-				'title' => '',
-
-			)
-		);
-
-		$this->buttons[] = '<a title="' . esc_attr( $args['title'] ) . '" id="' . esc_attr( $args['id'] ) . '" class="button ' . esc_attr( $args['class'] ) . '" href="' . esc_url( $args['url'] ) . '">' . esc_html( $args['value'] ) . '</a>';
-	}
-
-	/**
-	 * Showing the buttons
-	 */
-	private function show_buttons() {
-		if ( $this->has_fb_admins() ) {
-			$this->add_button(
-				array(
-					'url'   => add_query_arg( array(
-						'nonce'      => wp_create_nonce( 'fbclearall' ),
-						'fbclearall' => 'true',
-					), admin_url( $this->admin_url . '#top#facebook' ) ),
-					'value' => __( 'Clear all Facebook Data', 'wordpress-seo' ),
-				)
-			);
-		}
-
-		if ( is_array( $this->buttons ) && $this->buttons !== array() ) {
-			echo '<p class="fb-buttons">' . implode( '', $this->buttons ) . '</p>';
-		}
-
-		return $this;
-	}
-
-	/**
-	 * Check if the clear button should be displayed. This is based on the set options.
-	 *
-	 * @return bool When fb admins is a valid array.
-	 */
-	private function has_fb_admins() {
-		return is_array( $this->fb_admins ) && $this->fb_admins !== array();
-	}
-
-	/**
-	 * Creates nonce for removal link
-	 *
-	 * @return mixed
-	 */
-	private function get_delete_nonce() {
-		return wp_create_nonce( 'delfbadmin' );
-	}
-}
```

Comment:
This file is still loaded in vendor/composer/autoload_classmap.php & vendor/composer/autoload_static.php

Reply:
A `composer du` fixed this.